tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Don't commit `tsconfig.json`! If you need it, then symlink it from `tasegir` using `ln -s ./node_modules/tasegir/src/config/tsconfig.json ./tsconfig.json`. `experimentalDecorators` are present there...